cmake/Modules/SetCXX2011Flag.cmake

```diff
@@ -29,11 +29,13 @@
 
 include(CheckCXXCompilerFlag)
 
-# Set support for C++ 2011
-CHECK_CXX_COMPILER_FLAG("-std=c++11" COMPILER_SUPPORTS_CXX11)
```

Comment:
This is omitted, which is causing the Travis builds to fail. We must bring `CHECK_CXX_COMPILER_FLAG` back.